src/Microsoft.PowerFx.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 17
-VisualStudioVersion = 17.0.31717.71
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.32428.217
```

Comment:
>16.0.32428.217 [](http://example.com/codeflow?start=22&length=14)

Version stamps in source control? :grinning:

Reply:
should I revert? This is just VS being VS.

Reply:
Doesn't really matter, imo. Fine to leave as is.